java/test/jmri/jmrix/loconet/pr4/PR4AdapterTest.java

```diff
@@ -23,15 +23,19 @@ public void testcommandStationOptions() {
         PR4Adapter t = new PR4Adapter();
         String[] cmdStns = t.commandStationOptions();
         boolean foundPR4StandaloneProgrammer = false;
+        boolean foundPR4StandaloneLocoNet = false;
         for (int i=0; i < cmdStns.length; i++) {
-            Assert.assertNotEquals("should not find 'Stand-alone LocoNet", 
-                    LnCommandStationType.COMMAND_STATION_STANDALONE.getName(), cmdStns[i]);
             if (cmdStns[i].equals(LnCommandStationType.COMMAND_STATION_PR4_ALONE.getName())) {
                 foundPR4StandaloneProgrammer = true;
             }
+            if (cmdStns[i].compareTo(
+                    LnCommandStationType.COMMAND_STATION_STANDALONE.getName().toString() +
+                    " (using external LocoNet Data Termination!)") == 0) {
```

Comment:
Shouldn't this string be in the properties file so it could be translated?

Reply:
No, because this string text is stored in the configuration profile, and info in the config  profile must _not_ be I18N'ed.

Reply:
It would be good if these strings are marked with `// NOI18N` if they shouldn't be I18N'ed.